proposals/open-telemetry/microprofile-opentelemetry-tracing.asciidoc

```diff
@@ -0,0 +1,190 @@
+:authors: Roberto Cortez
+:email: radcortez@yahoo.com
+:version-label!:
+:sectanchors:
+:doctype: book
+:license: Eclipse Foundation Specification License v1.0
+:source-highlighter: coderay
+:toc: left
+:toclevels: 4
+:sectnumlevels: 4
+ifdef::backend-pdf[]
+:pagenums:
+endif::[]
+
+= Eclipse MicroProfile OpenTelemetry Tracing
+
+== Introduction
+
+TODO
+
+== Rationale
+
+TODO
+
+== Architecture
+
+This specification defines an easy way to allow an application running in a cloud native container to take advantage of
+distributed tracing by using the https://opentelemetry.io[OpenTelemetry project] (a merger of the OpenTracing and
+OpenCensus projects).
+
+This document and implementations *MUST* comply with the following OpenTelemetry specifications:
+
+- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md[OpenTelemetry Overview]
+(except for Metrics)
+- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md[Tracing API]
+- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md[Baggage API]
+- https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/context[Context API]
+- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md[Resource SDK]
+
+=== Automatic Instrumentation
+
+JAX-RS applications, JAX-RS Clients, and MicroProfile REST Clients are automatically enlisted to participate in
+distributed tracing without code modification.
```

Comment:
Do we want to specify how to use the `SpanKind`?  The default span kind is just `Internal`.  Without a standard set of span attributes, do we expect implementation from different vendors can create spans to form a single trace?

Reply:
The `SpanKind` and span attributes are specified in the OTel semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions. 

This is referred in https://github.com/eclipse/microprofile-sandbox/pull/89/files#diff-104c86912f426a8304aea09399aeb4822f91b34e0fb5ea7616c9740faf377925R162

Reply:
Added link to the Semantic Conventions section.